### PR TITLE
Fixed links

### PR DESCRIPTION
**Pull request by rosscg**
**By submitting this pull request, I confirm the following:** 
*please fill any appropriate checkboxes, e.g: [X]*

- [ x] I have read and understood the [contributors guide](https://github.com/pi-hole/pi-hole/blob/master/CONTRIBUTING.md), as well as this entire template.
- [x ] I have made only one major change in my proposed changes.
- [ x] I have commented my proposed changes within the code.
- [ x] I have tested my proposed changes, and have included unit tests where possible.
- [ x] I am willing to help maintain this change if there are issues with it later.
- [ x] I give this submission freely and claim no ownership.
- [ x] It is compatible with the [EUPL 1.2 license](https://opensource.org/licenses/EUPL-1.1)
- [ x] I have squashed any insignificant commits. ([`git rebase`](http://gitready.com/advanced/2009/02/10/squashing-commits-with-rebase.html))

Please make sure you [Sign Off](https://github.com/pi-hole/pi-hole/wiki/How-to-signoff-your-commits.) all commits. Pi-hole enforces the [DCO](https://github.com/pi-hole/pi-hole/wiki/Contributing-to-the-project).

---
**What does this PR aim to accomplish?:**
*A detailed description, screenshots (if necessary), as well as links to any relevant GitHub issues*

**How does this PR accomplish the above?:**
*A detailed description (such as a changelog) and screenshots (if necessary) of the implemented fix*

**What documentation changes (if any) are needed to support this PR?:**
*A detailed list of any necessary changes*

---
* You must follow the template instructions. Failure to do so will result in your pull request being closed.
* Please respect that Pi-hole is developed by volunteers, who can only reply in their spare time.

### CODE REVIEW

File: docs/guides/vpn/overview.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ Via this VPN, you can:
 - access your admin interface remotely
 - encrypt your Internet traffic
 
-If you don't want a full-tunnel, we provide a page of how to [set up your server to exclusively route DNS traffic, but nothing else via the VPN](only-dns-via-vpn.md). On another optional page, we describe how to set up Pi-hole + VPN in such a way that it is [usable both locally (no VPN) and from remote (through VPN)](dual-operation.md) while preserving full functionality.
+If you don't want a full-tunnel, we provide a page of how to [set up your server to exclusively route DNS traffic, but nothing else via the VPN](vpn/only-dns-via-vpn.md). On another optional page, we describe how to set up Pi-hole + VPN in such a way that it is [usable both locally (no VPN) and from remote (through VPN)](vpn/dual-operation.md) while preserving full functionality.
```
Comment (dschaper) [inline docs/guides/vpn/overview.md:17]: Same here, there are no `vpn/only-dns-via-vpn.md` files.

**Review by dschaper — CHANGES_REQUESTED** [review]

File: docs/guides/vpn/overview.md [hunk 2]
```diff
@@ -1,4 +1,4 @@
->This tutorial is tailored for setting up OpenVPN on a cloud-hosted virtual server (such as [Digital Ocean](https://www.digitalocean.com/?refcode=344d234950e1)). If you wish to have this working on your home network, you will need to tailor Pi-hole to listen on `eth0` (or similar), which we explain in [this section of the tutorial](dual-operation.md).
+>This tutorial is tailored for setting up OpenVPN on a cloud-hosted virtual server (such as [Digital Ocean](https://www.digitalocean.com/?refcode=344d234950e1)). If you wish to have this working on your home network, you will need to tailor Pi-hole to listen on `eth0` (or similar), which we explain in [this section of the tutorial](vpn/dual-operation.md).
```
[the remark this answers is not in the document]
Comment (PromoFaux) [reply docs/guides/vpn/overview.md:2]: Just checked on the main docs page.. the links seem to be OK anyway...

image


image

[the remark this answers is not in the document]
Comment (dschaper) [reply docs/guides/vpn/overview.md:2]: Which link seems to be okay?

[the remark this answers is not in the document]
Comment (PromoFaux) [reply docs/guides/vpn/overview.md:2]: All the ones the PR is seemingly trying to address. I'm not sure what is broken about them that needs fixing.

[the remark this answers is not in the document]
Comment (PromoFaux) [reply docs/guides/vpn/overview.md:2]: things that work

[the remark this answers is not in the document]
Comment (rosscg) [reply docs/guides/vpn/overview.md:2]: You are right, I'm not sure what was causing the problem I experienced as I now cannot replicate it.

**Comment by rosscg** [comment]
You are right, I'm not sure what was causing the problem I experienced as I now cannot replicate it.

**Pull request closed without merging**